docs/admin/deployment.md

```diff
@@ -1015,64 +1015,67 @@ the necessary DAOS metadata indicating that the server has been formatted.
 When starting, `daos_server` will skip `maintenance mode` and attempt to start
 I/O engines if valid DAOS metadata is found in `scm_mount`.
 
+
 ## Agent Setup
 
-This section addresses how to configure the DAOS agents on the storage
-nodes before starting it.
+This section addresses how to configure the DAOS Agents on the client nodes
+(and optionally on the DAOS server nodes).
+
+### Agent User and Group Setup
+
+The `daos_agent` daemon runs as an unprivileged user process,
+with the username `daos_agent` and group name `daos_agent`.
+If those IDs do not exist at the time the `daos-client` RPM is installed,
+they will be created during the RPM installation.  Refer to
+[User and Group Management](https://docs.daos.io/v2.0/admin/predeployment_check/#user-and-group-management)
+for details.
 
 ### Agent Certificate Generation
 
-The DAOS security framework relies on certificates to authenticate
-administrators. The security infrastructure is currently under
-development and will be delivered in DAOS v1.0. Initial support for certificates
-has been added to DAOS and can be disabled either via the command line or in the
-DAOS Agent configuration file. Currently, the easiest way to disable certificate
-support is to pass the -i flag to `daos_agent`.
+The DAOS security framework relies on SSL certificates to authenticate
+the DAOS daemon processes, as well as the DAOS administrators who run the
+`dmg` command.
+Refer to [Certificate Generation](https://docs.daos.io/v2.0/admin/deployment/?h=gen_#certificate-configuration)
+for details on creating the necessary certificates.
+
+It is possible to disable the use of certificates for testing purposes.
+This is strongly discouraged in production environments, as it will allow
```

Comment:
Stronger wording than this is probably justified. DAOS should never be run without the certs in production, for the reasons cited.

Reply:
fixed